app/mailers/planning_application_mailer.rb

```diff
@@ -32,12 +32,11 @@ def validation_notice_mail(planning_application, host, user)
   def invalidation_notice_mail(planning_application, host)
     @host = host
     @planning_application = planning_application
-    @application_accountable_email = @planning_application.applicant_and_agent_email.first
```

Comment:
why remove this ?

Reply:
instead of calling this method repeatedly, I've replaced it directly on the view_mail method:
```
  to: @planning_application.applicant_and_agent_email.first,
  ```
  
  ```
    def invalidation_notice_mail(planning_application, host)
    @host = host
    @planning_application = planning_application

    view_mail(
      NOTIFY_TEMPLATE_ID,
      subject: "Your planning application is invalid",
      to: @planning_application.applicant_and_agent_email.first,
      reply_to_id: @planning_application.local_authority.reply_to_notify_id
    )
  end
  ```